src/index.js

```diff
@@ -424,9 +441,12 @@ class ServerlessLayers {
 
     if (!outputs) return null;
 
-    const logicalId = this.getOutputLogicalId(this.getLayerName());
+    const layerName = this.slsLayersConfig.shouldUseLayersArtifactory ? this.slsLayersConfig.artifactoryLayerName : this.getLayerName();
+    const exportName = this.getOutputLogicalId(layerName);
 
-    const arn = (outputs.find(x => x.OutputKey === logicalId) || {}).OutputValue;
+    console.log(`[ LayersPlugin ]: going to try and find if layer name ${layerName} with the export name ${exportName} is in outputs ${JSON.stringify(outputs)}`);
```

Comment:
sometimes it's better to add `null, 2` to `JSON.stringify` to make it more readable: `${JSON.stringify(outputs, null, 2)})`. but if you see the logs are readable the way it is that's ok just FYI